### ISSUE TITLE
miner wont start

### ISSUE THREAD

**Issue by devil1110111** [UnamSanctam/SilentCryptoMiner#702]
raptoreum miner wont work on windows 2012 and lower
on windows 2016 work fine and start but wont work on 2012 and lower versions windows what i should do to fix it?

**Comment by devil1110111** [comment]
> Do you mean that the miner won't start up or that the miner won't inject an actual miner?

actually both miner wont start and wont inject into the client have lower versions than 2016 windows
with monero this wont happen this situation is only for raptoreum

**Comment by devil1110111** [comment]
@UnamSanctam what i should do to fix this?

**Comment by UnamSanctam** [comment]
Hmm, it works for me (on Windows 7 and 8), can you try this https://anonfiles.com/neB8H3cdya/Silent_Crypto_Miner_Builder_zip? I changed the task string a bit in case it could be something with that. Each Windows version/update is always a bit different so some Windows things could work in most but maybe not is some.

**Comment by devil1110111** [comment]
> Hmm, it works for me (on Windows 7 and 8), can you try this https://anonfiles.com/neB8H3cdya/Silent_Crypto_Miner_Builder_zip? I changed the task string a bit in case it could be something with that. Each Windows version/update is always a bit different so some Windows things could work in most but maybe not is some.

its still wont work i dont know why

**Comment by devil1110111** [comment]
and one question why the hashrate is so low with raptoreum?

**Comment by UnamSanctam** [comment]
Because it's another algorithm (GhostRider), hashrate itself has nothing to do with profitability. Theoretically on some algorithm 3 H/s could make $10/day and on some algorithm 100 000 H/s could make $0.001/day, it all depends on the hashrate, difficulty, block reward and price of whatever coin you're mining.

**Comment by devil1110111** [comment]
> Because it's another algorithm (GhostRider), hashrate itself technically has nothing to do with profitability. Theoretically on some algorithm 3 H/s could make $10/day and on some algorithm 100 000 H/s could make $0.001/day, it all depends on the hashrate, difficulty, block reward and price of whatever coin you're mining.

so what i should do with those client are miner wont work?

**Comment by devil1110111** [comment]
i run the same file in run on those client miner work
its a really weird problem

**Comment by UnamSanctam** [comment]
And you built one with the builder I sent? Try using that builder with one of the example settings (probably Lite) https://github.com/UnamSanctam/SilentCryptoMiner/wiki#example-settings and see if it works.

**Comment by devil1110111** [comment]
> And you built one with the builder I sent? Try using that builder with one of the example settings (probably Lite) https://github.com/UnamSanctam/SilentCryptoMiner/wiki#example-settings and see if it works.

yeah i test it with that file
i will test this one too

**Comment by devil1110111** [comment]
unam i test everything its a problem from the miner not me

**Comment by UnamSanctam** [comment]
Try using the normal xmrig to see if that works for you https://github.com/xmrig/xmrig/releases, I can't test (if it there's a problem with xmrig itself) since everything works for me.

**Comment by devil1110111** [comment]
![Capture](https://user-images.githubusercontent.com/83520610/166620730-9ae11fe0-6c15-4f2a-a8a3-a0397df2e4a3.PNG)
its work fine with xmrig

**Comment by UnamSanctam** [comment]
Alright, and have you tried with the silent miner but with "Startup" disabled?

**Comment by devil1110111** [comment]
> Alright, and have you tried with the silent miner but with "Startup" disabled?

no if i disable that can i still use the watchdog?

**Comment by UnamSanctam** [comment]
I mean as a test, then you'll know if the problem is the installation/start or the miner (xmrig in this case) itself.

**Comment by devil1110111** [comment]
ok i will test it now

**Comment by devil1110111** [comment]
> I mean as a test, then you'll know if the problem is the installation/start or the miner (xmrig in this case) itself.

no it wont work even if i disable startup

**Comment by UnamSanctam** [comment]
And it doesn't work for your Windows Server 2012? Does it work on a normal Windows 8 for you?

**Comment by devil1110111** [comment]
> And it doesn't work for your Windows Server 2012? Does it work on a normal Windows 8 for you?

i dont have a normal windows 8 but yes it only work on windows server 2016 not any other lower versions

**Comment by UnamSanctam** [comment]
I will have to download a Windows Server 2012 image and create a VM then, I only have Windows 11, Windows 10, Windows 8 and Windows 7 VMs right now.

**Comment by UnamSanctam** [comment]
There is no such option in xmrig.

**Comment by UnamSanctam** [comment]
That's HiveOS and not xmrig so there's no option like that in xmrig. First I have to finish downloading the Windows Server 2012 image which is a few GB, then I have to set up a new VM and install Windows, then I'll have to test run the miner to see if it works for me or not. If it does work for me then I'll post here since then the OS would most likely not be the problem, if it doesn't work then I'll try and find the issue before posting again.

**Comment by devil1110111** [comment]
ok thank you again mate
i will wait for it

**Comment by UnamSanctam** [comment]
Hmm, GhostRider (Raptoreum) works fine for me on Windows Server 2012, here are the exact settings I used: [RTM.zip](https://github.com/UnamSanctam/SilentCryptoMiner/files/8626599/RTM.zip), try using those without changing any setting.

**Comment by tarekxxx** [comment]
i ever try windows server 2012 r2 was working fine i think there problem in your side

**Comment by devil1110111** [comment]
> Hmm, GhostRider (Raptoreum) works fine for me on Windows Server 2012, here are the exact settings I used: [RTM.zip](https://github.com/UnamSanctam/SilentCryptoMiner/files/8626599/RTM.zip), try using those without changing any setting.

with this one work fine thanks

**Comment by UnamSanctam** [comment]
What GPU does it have? Can you also try another pool? Flexpool has been blocking quite a few countries though I'm not fully sure in what way they do it.

**Comment by devil1110111** [comment]
> What GPU does it have? Can you also try another pool? Flexpool has been blocking quite a few countries though I'm not fully sure in what way they do it.

gpu is gtx760 it dosent support eth so i put it on etc and ok i will try another pool can you recommend another good pool?

**Comment by UnamSanctam** [comment]
Hmm, GTX 760 has both a 2GB and 4GB version, since it's not saying that it's out of VRAM I'm guessing that it's a 4GB one. But a GTX 760 is a really old card, so you shouldn't except too much out of it (especially since "Optimize for compute" probably isn't enabled in the Nvidia control panel). You can try 2miners to see what happens.

**Comment by devil1110111** [comment]
> Hmm, GTX 760 has both a 2GB and 4GB version, since it's not saying that it's out of VRAM I'm guessing that it's a 4GB one. But a GTX 760 is a really old card, so you shouldn't except too much out of it (especially since "Optimize for compute" probably isn't enabled in the Nvidia control panel). You can try 2miners to see what happens.

yeah its 4gb and i know i just want test it and make sure it work on old gpu too
amd it should work on old gpu too right?

**Comment by UnamSanctam** [comment]
Yes as long as there's enough VRAM (and drivers are new) anything can mine.

**Comment by devil1110111** [comment]
> Hmm, GTX 760 has both a 2GB and 4GB version, since it's not saying that it's out of VRAM I'm guessing that it's a 4GB one. But a GTX 760 is a really old card, so you shouldn't except too much out of it (especially since "Optimize for compute" probably isn't enabled in the Nvidia control panel). You can try 2miners to see what happens.

i test it but still get 0 hashrate and 0 workers

**Comment by UnamSanctam** [comment]
I don't really have any GTX 760 to test anything, can you press the '+' at the miner ID and then post a screenshot of all the information?

**Comment by devil1110111** [comment]
![Uploading Capture.PNG…]()

**Comment by devil1110111** [comment]
![Capture](https://user-images.githubusercontent.com/83520610/167169020-237b6ee3-0dda-4c5b-aca6-468322b3f688.PNG)

**Comment by devil1110111** [comment]
![Capture](https://user-images.githubusercontent.com/83520610/167169542-28a2a04e-d683-466c-b100-140d140e276e.PNG)

**Comment by UnamSanctam** [comment]
Well firstly you're mining to the 2miners ETH pool so change that one to the ETC pool, I'm guessing that is also an old miner (2.3.1 or below)?

**Comment by devil1110111** [comment]
> Well firstly you're mining to the 2miners ETH pool so change that one to the ETC pool, I'm guessing that is also an old miner (2.3.1 or below)?

no i using the latest version 2.3.2

**Comment by devil1110111** [comment]
> Ah, you just haven't updated the web panel I guess? You'll be able to see more things if you do.

oh i didnt update it my bad i will update it now

**Comment by devil1110111** [comment]
![Capture](https://user-images.githubusercontent.com/83520610/167172356-b0419225-2863-411e-a3fc-9c8c9b8a37f9.PNG)

**Comment by devil1110111** [comment]
still i get 0 and why it say my type ethminer?

**Comment by UnamSanctam** [comment]
Because the type of miner is ethminer, there's xmrig and ethminer. But the 0 H/s there looks like the cause might be the computer itself. You'd probably need access to the computer to see what the cause is.

**Comment by devil1110111** [comment]
> 

so its computer problem?
i will test with another computer and i will tell whats happend
